### PR TITLE
Fix translation of user exists error

### PR DESCRIPTION
**Pull request by nickvergessen**
Before | After
---|---
Bildschirmfoto vom 2022-07-13 13-06-28 | Bildschirmfoto vom 2022-07-13 12-58-37

### CODE REVIEW

**Comment by nickvergessen** [comment]
/backport ac59b01be5f9a51ae78d754d7442cbdf087c9ddb to stable24

**Comment by nickvergessen** [comment]
/backport ac59b01be5f9a51ae78d754d7442cbdf087c9ddb to stable23

**Comment by nickvergessen** [comment]
/backport ac59b01be5f9a51ae78d754d7442cbdf087c9ddb to stable22

File: apps/settings/src/store/users.js [hunk 1]
```diff
@@ -552,6 +553,17 @@ const actions = {
 				.then((response) => dispatch('addUserData', userid || response.data.ocs.data.id))
 				.catch((error) => { throw error })
 		}).catch((error) => {
+			let statusCode = 0
+			try {
+				statusCode = error.response.data.ocs.meta.statuscode
+			} catch (e) {
+			}
```
Comment (Pytal) [inline apps/settings/src/store/users.js:560]:
```suggestion
			const statusCode = error.response.data.ocs?.meta?.statuscode
```
**suggestion (non-blocking)**

File: apps/settings/src/store/users.js [hunk 2]
```diff
@@ -552,6 +553,17 @@ const actions = {
...
+
+			if (statusCode === 102) {
+				showError(t('settings', 'User already exists.'), { isHTML: true })
+				throw error
```
Comment (Pytal) [inline apps/settings/src/store/users.js:564]:
```suggestion
```
Already thrown below

File: apps/settings/src/store/users.js [hunk 3]
```diff
@@ -552,6 +553,17 @@ const actions = {
...
 				.catch((error) => { throw error })
 		}).catch((error) => {
+			let statusCode = 0
+			try {
+				statusCode = error.response.data.ocs.meta.statuscode
+			} catch (e) {
+			}
+
+			if (statusCode === 102) {
+				showError(t('settings', 'User already exists.'), { isHTML: true })
```
Comment (Pytal) [inline apps/settings/src/store/users.js:563]:
```suggestion
				showError(t('settings', 'User already exists.'))
```
Non-HTML

File: apps/settings/src/store/users.js (hunk 2 above)
Comment (nickvergessen) [reply apps/settings/src/store/users.js:564]: But then a second showError() is triggered

**Review by CarlSchwan — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by nickvergessen** [comment]
/backport ba9815fbe1abfaf87bab8453929dbfd9538d2d6c to stable23

**Comment by nickvergessen** [comment]
/backport ba9815fbe1abfaf87bab8453929dbfd9538d2d6c to stable22

**Pull request merged**